package.json

```diff
@@ -14,6 +14,7 @@
     "publish:dry": "nx run-many -t publish --output-style=static --dry",
     "e2e": "nx run-many -t e2e --output-style=stream --skipInstall",
     "e2e:ci": "nx run-many -t e2e --configuration=production --output-style=stream --skipInstall",
+    "e2e:overlay:ci": "nx run-many -t e2e --configuration=productionOverlay --output-style=stream --skipInstall",
```

Comment:
maybe it's better to extend previous command in nx to have single point of e2e starts?

Reply:
thank you, resolved